packages/cli/src/metro-config.ts

```diff
@@ -130,6 +131,13 @@ export function customizeMetroConfig(
   const metroConfig = metroConfigReadonly as InputConfigT;
 
   const plugins: MetroPlugin[] = [];
+  if (customSerializationPlugins) {
+    customSerializationPlugins.forEach((plugin) => {
+      const implPath = require.resolve(plugin.path, { paths: [process.cwd()] });
```

Comment:
Should this be relative to project root?
```suggestion
      const implPath = require.resolve(plugin.path, { paths: [metroConfig.projectRoot] });
```